agir/groups/components/messages/PromoMessageModal.js

```diff
@@ -0,0 +1,282 @@
+    cursor: pointer;
+  }
+
+  header {
+    position: relative;
+    width: 100%;
+    height: 177px;
+    background-color: ${style.secondary500};
+    background-position: bottom center;
+    background-repeat: no-repeat;
+    margin-bottom: 56px;
+
+    @media (max-width: ${style.collapse}px) {
+      height: 100px;
+      background-size: 160px auto;
+    }
+
+    &::after {
+      content: "";
+      display: block;
+      background-color: transparent;
+      background-repeat: no-repeat;
+      background-size: cover;
+      width: 100px;
+      height: 100px;
+      position: absolute;
+      bottom: 0;
+      left: 50%;
+      transform: translate3d(-50%, 50%, 0);
+    }
+  }
+`;
+
+const Container = styled.div`
+  display: inline-flex;
+  justify-content: center;
+  align-items: center;
+  flex-direction: column;
+  padding: 2rem;
+  overflow-y: auto;
+  width: 100%;
+  border: 1px solid #dfdfdf;
+  background-color: #fff;
+
+  p {
+    margin: 30px 0;
+    font-size: 18px;
+    text-align: center;
+    max-width: 600px;
+  }
+
+  img {
+    user-select: none;
+  }
+
+  @media (max-width: ${style.collapse}px) {
+    padding: 1rem;
+    img {
+      width: 150px;
+    }
+  }
+`;
+
+const StyledLabel = styled.div`
+  color: ${style.redNSP};
+`;
+
+const Title = styled.div`
+  margin: 0.5rem 0;
+  font-size: 1.5rem;
+  font-weight: 700;
+`;
+
+const Arrow = styled.div`
+  border: 1px solid #dfdfdf;
+  border-radius: 100px;
+  display: inline-flex;
+  padding: 0.5rem;
+  cursor: pointer;
+  user-select: none;
+`;
+
+const Mark = styled.span`
+  width: 0.5rem;
+  height: 0.5rem;
+  margin: 0.188rem;
+  display: inline-block;
+  border-radius: 2rem;
+  cursor: pointer;
+  transition: background-color 0.5s ease-in-out;
+  background-color: ${(props) =>
+    props.$active ? style.black700 : style.black200};
+`;
+
+const items = [
+  {
+    img: promo1,
+    content: (
+      <p>
+        <strong>Lancez une première conversation dans votre groupe ! </strong>
+        Discustez de vos prochaines actions sur Action Populaire
+      </p>
+    ),
+  },
+  {
+    img: promo2,
+    content: (
+      <p>
+        <strong>Joignez simplement vos membres : </strong>
+        tous vos membres recevront un e-mail avec le contenu de votre message et
+        pourront y répondre sur le site
+      </p>
+    ),
+  },
+  {
+    img: promo3,
+    content: (
+      <p>
+        <strong>Organisez ensemble vos actions : </strong>
+        les membres du groupes sont les seuls à voir et commenter
+      </p>
+    ),
+  },
+  {
+    img: promo4,
+    content: (
+      <p>
+        <strong>Restez connecté·es ! </strong>
+        Recevez les notifications de message en téléchargeant l’application
+      </p>
+    ),
+  },
+];
+
+export const PromoMessage = (props) => {
+  const { onClick, onClose, goToMessages } = props;
+
+  const [itemIndex, setItemIndex] = useState(0);
+
+  const handleNext = () => {
+    setItemIndex((state) => (state + 1) % items.length);
```

Comment:
```suggestion
    setItemIndex((itemIndex + 1) % items.length);
```

et de même pour les autres non ?